cob_people_detection/ros/launch/people_detection_display.launch

```diff
@@ -1,17 +1,21 @@
 <?xml version="1.0"?>
 
 <launch>
+  
+  <arg name="display_results_with_image_view" default="true"/>   <!-- set to false if you do not like to display the camera image with names attached to detected faces within an image_view window -->
+  <arg name="face_positions_topic_name_in" value="/face_detector/face_positions"/>
```

Comment:
Did you test this? Does it work with "/" at the beginning? Before it was without.